src/index.html

```diff
@@ -1,12 +1,353 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="page__header header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="Moyo-logo">
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#!" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+
+          <li class="nav__item">
+            <a href="#!" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+
```

Comment:
just one empty line)
```suggestion
      <a href="#home" class="header__logo">
        <img src="./images/logo.png" alt="Moyo-logo">
      </a>
      
      <nav class="header__nav nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#!" class="nav__link is-active">
              Apple
            </a>
          </li>

          <li class="nav__item">
            <a href="#!" class="nav__link">
              Samsung
            </a>
          </li>

```